### PR TITLE
Loki: Rename public/app/plugins/datasource/loki/live_streams_result_transformer.ts file to follow convention

### LINKED ISSUE #54648: Loki: Rename public/app/plugins/datasource/loki/live_streams_result_transformer.ts file to follow convention
**Level: easy**

Some files that are maintained by logs squad have snake_case name. According to convention https://github.com/grafana/grafana/blob/765b995b1bcafe8b94d0bbe53737ea6c751d63c9/contribute/style-guides/frontend.md#file-and-directory-naming-conventions we should use:
- `camelCase` when the primary export is a function
- `PascalCase` when the primary export is a class or React component. 

Rename following files  in `grafana/grafana` to follow the above-mentioned convention:
- `public/app/plugins/datasource/loki/live_streams_result_transformer.ts`
- `public/app/plugins/datasource/loki/live_streams_result_transformer.test.ts`

### PR DESCRIPTION
**Pull request by gitstart**
**What this PR does / why we need it**:
Change the name and the imports of "live_streams_result_transformer" and "live_streams_result_transformer.test" to follow the convention

**Which issue(s) this PR fixes**:

Fixes #54648

**Special notes for your reviewer**:

### CODE REVIEW

File: .betterer.results [hunk 1]
```diff
@@ -6939,7 +6939,16 @@ exports[`better eslint`] = {
     "public/app/plugins/datasource/loki/getDerivedFields.ts:5381": [
       [0, 0, 0, "Unexpected any. Specify a different type.", "0"],
       [0, 0, 0, "Do not use any type assertions.", "1"]
+
+    "public/app/plugins/datasource/loki/language_provider.ts:5381": [
+      [0, 0, 0, "Unexpected any. Specify a different type.", "0"],
+      [0, 0, 0, "Unexpected any. Specify a different type.", "1"],
+      [0, 0, 0, "Unexpected any. Specify a different type.", "2"]
     ],
```
Comment (ivanahuckova) [inline .betterer.results:6947]: Not sure where is this coming from. 🤔 Would you try to merge main branch to see if it fixes itself?

**Review by ivanahuckova — COMMENTED** [review]
Looks good! I left the comment bellow related to updates in `.betterer.results` file.

**Comment by gitstart** [comment]
@ivanahuckova Thank you for the review, the issue has been resolve please take a look

**Review by ivanahuckova — APPROVED** [review]
LGTM! Thanks for your contribution.

**Pull request merged**